app/javascript/controllers/keywords_controller.js

```diff
@@ -23,16 +23,23 @@ export default class extends Controller {
     this.containerTarget.classList.remove('is-invalid')
   }
 
-  checkForEnter(event) {
-    if (event.key === 'Enter') {
-      event.preventDefault() // prevent form submission
-      this.add(event);
```

Comment:
i spotted this similar method to what we just implemented for sunetids, but could not find any references to this particular "checkForEnter()" method anywhere in the code, so I removed it